pages/QuizCard/index.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { StyledDiv } from "../../components/QuizCardPage/QuizCard";
+import { StyledForm } from "../../components/QuizCardPage/QuizPageForm";
+import { StyledDiv2 } from "../../components/QuizCardPage/QuizCard";
+import { StyledUser1 } from "../../components/QuizCardPage/User";
+import { StyledUser2 } from "../../components/QuizCardPage/User";
+import { StyledPoints1 } from "../../components/QuizCardPage/UserPoints";
+import { StyledPoints2 } from "../../components/QuizCardPage/UserPoints";
+import { StyledHeader } from "../../components/QuizCardPage/Header";
+import { StyledCard } from "../../components/QuizCardPage/QuizCard";
+import { StyledTimer } from "../../components/QuizCardPage/Timeline";
+import { StyledDiv3 } from "../../components/QuizCardPage/QuizCard";
+import { StyledButton } from "../../components/QuizCardPage/AnswerButton";
+import { StyledDiv4 } from "../../components/QuizCardPage/QuizCard";
+import { StyledDiv5 } from "../../components/QuizCardPage/QuizCard";
+import { StyledButtonNext } from "../../components/QuizCardPage/AnswerButton";
+import { StyledLabel } from "../../components/QuizCardPage/User";
+const questions = [
+  {
+    id: 1,
+    question: "What is the capital of France?",
+    answers: ["Paris", "Berlin", "Madrid", "London"],
+    correctAnswer: "Paris",
+  },
+];
+
+export default function QuizCard() {
+  // Für die Fragen
+  const [questionsState, setQuestionsState] = useState(questions);
+  // Für die Zeit die abläuft
+  const [timeLeft, setTimeLeft] = useState(30);
+  // Für die Punkte der User
+  const [points1, setPoints1] = useState(0);
+  const [points2, setPoints2] = useState(0);
+
+  // useEffect benutzt für Timer runter zählen
+  useEffect(() => {
+    if (timeLeft > 0) {
+      const intervalId = setInterval(() => {
+        setTimeLeft((prevTimeLeft) => prevTimeLeft - 1);
+      }, 1000);
+
+      return () => {
+        clearInterval(intervalId);
+      };
+    }
+  }, [timeLeft]);
+
+  return (
+    <StyledDiv>
+      <StyledForm>
```

Comment:
Your form is missing the submit button because the `StyledButton`s are `submit` by default and submit your form; this is why your "Weiter" Button is never displayed and the points of user1 is only displayed for a second before the page reloads.